File: 프로그래머스/unrated/120899. 가장 큰 수 찾기/가장 큰 수 찾기.py
def solution(array):
    #max 함수를 사용하여 배열에서 가장 큰수를 찾습니다.
    max_value = max(array)
    #index 메서드를 사용하여 가장 큰수의 인덱스를 찾습니다.
    max_index = array.index(max_value)
    #가장 큰수와 그 수의 인덱스를 담은 배열을 반환합니다.
    answer = [max_value, max_index]
    return answer

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
        int[] answer = new int[2];
        int max=0;
        int maxarray=0;
        for(int i=0;i<array.length;i++){
            if(array[i]>max){
                max=array[i];
                maxarray=i;
            }
        }
        answer[0]=max;
        answer[1]=maxarray;
        return answer;
    }
}